README.md

```diff
@@ -1 +1,16 @@
-# Lavinia-client
\ No newline at end of file
+# Lavinia-client
+This guide in other languages:
+* [Norsk bokmål](README.nob.md)
+
+# Prerequisites
+You have installed:
+* [Node.js](https://nodejs.org)
+* [Yarn](https://yarnpkg.com)
+
+# Running the development build
+1. Clone the repository to your preferred folder
+2. Open your favourite shell and navigate to aforementioned folder
```

Comment:
I think we should call it command line interface, console or terminal, as people are most likely to use a CLI to execute these commands.